server/src/main/java/org/elasticsearch/transport/Transport.java

```diff
@@ -34,6 +36,8 @@
 
 public interface Transport extends LifecycleComponent {
 
+    Logger logger = LogManager.getLogger(Transport.class);
```

Comment:
This looks unused, but I haven't checked for certain. If it is used, can it be `private static final` please?